blurb/blurb.py

```diff
@@ -745,19 +745,18 @@ def test(*args):
 
 
 def find_editor():
-    for var in 'GIT_EDITOR', 'EDITOR':
-        editor = os.environ.get(var)
-        if editor is not None:
-            return editor
```

Comment:
The patch removes $EDITOR.  $EDITOR should be the first thing checked.

As I've already mentioned, $EDITOR has been a UNIX shell standard in excess of thirty years.  It should be the *first* place a tool looks for the user's preferred text editor.  I'm happy to accept a patch that adds uses 'git var GIT_EDITOR' in a sensible way, but blurb must check $EDITOR *first.*